go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/ethereum/evmc/v7
+
+go 1.14
```

Comment:
The readme say "Go (bindings) | 1.9 - 1.12". Is this changing that?

Reply:
you can still build it on the old golang, but go modules will require go 1.14

I can adjust README

Reply:
updated readme

Reply:
I'm not sure what this mean, but this works in Go 1.11 (the first version that supports modules).

Reply:
I changed that to `1.11` as some random comment on internet this is minimum Go version of the module.